.travis.yml

```diff
@@ -41,6 +43,10 @@ install:
       conda install --yes sphinx sphinx_rtd_theme nbsphinx sphinx-autodoc-typehints jupyter_client matplotlib;
       pip install sklearn;
     fi
+  - if [ "$JAX_TF_BRIDGE" = true ] ;then
+      # jax_to_tf needs some fixes that are not in tensorflow==2.2.0
+      pip install tf-nightly>=2.3.0.dev20200525 ;
```

Comment:
I would pin a fixed version to avoid having the CI fail when a broken version is pushed.

Reply:
Done